scene/resources/texture.cpp

```diff
@@ -82,12 +82,28 @@ void Texture2D::draw_rect_region(RID p_canvas_item, const Rect2 &p_rect, const R
 }
 
 bool Texture2D::get_rect_region(const Rect2 &p_rect, const Rect2 &p_src_rect, Rect2 &r_rect, Rect2 &r_src_rect) const {
+	bool ret = false;
+	if (GDVIRTUAL_CALL(_get_rect_region, ret, p_rect, p_src_rect, r_rect, r_src_rect)) {
```

Comment:
This method isn't declared as having a return value, did you mean to bind it as:
```cpp
GDVIRTUAL4RC(bool, _get_rect_region, Rect2, Rect2, Rect2, Rect2)
```

Also return value goes last:
```suggestion
	if (GDVIRTUAL_CALL(_get_rect_region, p_rect, p_src_rect, r_rect, r_src_rect, ret)) {
```